universal-login-example/src/relayer/TokenGrantingRelayer.js

```diff
@@ -20,11 +21,15 @@ class TokenGrantingRelayer extends Relayer {
           this.tokenContract.transfer(identityContract.address, utils.parseEther('100'));
         });
     });
-    this.addKeySubscription = this.hooks.addListener('added', (identityAddress) => {
-      this.tokenContract.transfer(identityAddress, utils.parseEther('5'));
+    this.addKeySubscription = this.hooks.addListener('added', (transaction) => {
+      waitToBeMined(this.provider, transaction.hash).then(() => {
```

Comment:
await

Reply:
check status and only send tokens if success

Reply:
✅